src/agent/srcview/src/pdbcache.rs

```diff
@@ -49,7 +52,11 @@ impl PdbCache {
 
             while let Some(symbol) = symbols.next()? {
                 if let Ok(SymbolData::Procedure(proc)) = symbol.parse() {
+                    let proc_name = proc.name.to_string();
                     let mut lines = program.lines_at_offset(proc.offset);
+
+                    let symbol_to_lines = symbol_to_lines.entry(proc_name.to_string()).or_default();
```

Comment:
Stylistic but `clone` instead of `to_string` to be explicit about what's happening, since it's already a `String`? And another instance of this below…

Reply:
@DrChat that's fine then, it won't be confusing in the IDE with type information